projects/golang/math_big_fuzzer.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+
+package mathfuzzer
+
+import "math/big"
+
+func FuzzCmp(data []byte) int {
+    if !isDivisibleBy(len(data), 2) {
+        return -1
+    }
+    i1 := new(big.Int)
+    i2 := new(big.Int)
+
+    half := len(data) / 2
+
+    halfOne := data[:half]
+    halfTwo := data[half:]
+
+    i1.SetBytes(halfOne)
+    i2.SetBytes(halfTwo)
+
+    i1.Cmp(i2)
+    return 1
+}
+
+func FuzzExpNN(data []byte) int {
```

Comment:
This currently isn't calling `big.Int.Exp`, so I think there's some code missing here.
Additionally, if you want to do that, this target should be renamed to `FuzzExp` instead of `FuzzExpNN`.

Reply:
We could include more in the fuzzer, but calling SetString will reach the previous target: https://github.com/golang/go/blob/master/src/math/big/ratconv.go#L172

Reply:
Sure sounds good.